wled00/bus_wrapper.h

```diff
@@ -612,6 +767,22 @@ class PolyBus {
       case I_32_I1_TM1_4: (static_cast<B_32_I1_TM1_4*>(busPtr))->SetPixelColor(pix, col); break;
       case I_32_I1_TM2_3: (static_cast<B_32_I1_TM2_3*>(busPtr))->SetPixelColor(pix, RgbColor(col.R,col.G,col.B)); break;
       #endif
+      case I_32_RN_UCS_3: (static_cast<B_32_RN_UCS_3*>(busPtr))->SetPixelColor(pix, Rgb48Color(col.R,col.G,col.B)); break;
+      #ifndef WLED_NO_I2S0_PIXELBUS
+      case I_32_I0_UCS_3: (static_cast<B_32_I0_UCS_3*>(busPtr))->SetPixelColor(pix, Rgb48Color(col.R,col.G,col.B)); break;
+      #endif
+      #ifndef WLED_NO_I2S1_PIXELBUS
+      case I_32_I1_UCS_3: (static_cast<B_32_I1_UCS_3*>(busPtr))->SetPixelColor(pix, Rgb48Color(col.R,col.G,col.B)); break;
+      #endif
+//      case I_32_BB_UCS_3: (static_cast<B_32_BB_UCS_3*>(busPtr))->SetPixelColor(pix, Rgb48Color(col.R,col.G,col.B)); break;
+      case I_32_RN_UCS_4: (static_cast<B_32_RN_UCS_4*>(busPtr))->SetPixelColor(pix, Rgbw64Color(col.R,col.G,col.B,col.W)); break;
+      #ifndef WLED_NO_I2S0_PIXELBUS
+      case I_32_I0_UCS_4: (static_cast<B_32_I0_UCS_4*>(busPtr))->SetPixelColor(pix, Rgbw64Color(col.R,col.G,col.B,col.W)); break;
+      #endif
+      #ifndef WLED_NO_I2S1_PIXELBUS
+      case I_32_I1_UCS_4: (static_cast<B_32_I1_UCS_4*>(busPtr))->SetPixelColor(pix, Rgbw64Color(col.R,col.G,col.B,col.W)); break;
+      #endif
```

Comment:
Hi. This code block doesn't work because `Rgbw64Color` has 16 bits parameters but `col` has 8 bits members, so the LED strip is always black.

If you want to use the dynamic conversion from 8 bits to 16 bits, `Rgbw64Color` has a constructor with a `RgbwColor`, so the working code is :

```cpp
case I_32_RN_UCS_4: (static_cast<B_32_RN_UCS_4*>(busPtr))->SetPixelColor(pix, Rgbw64Color(col)); break;
#ifndef WLED_NO_I2S0_PIXELBUS
case I_32_I0_UCS_4: (static_cast<B_32_I0_UCS_4*>(busPtr))->SetPixelColor(pix, Rgbw64Color(col)); break;
#endif
#ifndef WLED_NO_I2S1_PIXELBUS
case I_32_I1_UCS_4: (static_cast<B_32_I1_UCS_4*>(busPtr))->SetPixelColor(pix, Rgbw64Color(col)); break;
#endif
```

Reply:
It does work on my system.

Reply:
@blazoncek I don't know what is your system, but I can assure you that my LED strip is always black without my modification.
Thanks Makuna, I'm a newbie in WLED and NeoPixelBus but I tested extensively your library, enough to understand the problem of the 8bits -> 16bits conversion.